### DISCUSSION TITLE
Is there a way to decompile with Forge?

### DISCUSSION THREAD

**Discussion by complected** [foundry-rs/foundry#1734] [Help] [answered]
Like [this](https://ethervm.io/decompile/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). But this tool doesn't have source I think so I don't know what it's doing at all...

Is there a way with Forge? [`getCode`](https://github.com/foundry-rs/foundry/blob/5e20e3dadc07047517fa30fe496231207453e409/forge/README.md?plain=1#L171) doesn't look right but close since getCode returns bytecode for the contract source, I am looking for bytecode->opcode. 

Thanks in advance!

**Comment by onbjerg** [comment, marked as answer]
There is not currently, no.
